eox_tagging/test/test_viewset.py

```diff
@@ -180,3 +199,27 @@ def test_soft_delete(self):
         response = self.client.delete(URL)
 
         self.assertEqual(response.status_code, 204)
+
+    def test_retreive_invalid_tag(self):
+        """Used to test getting a tag given its key."""
+        KEY = self.example_tag.key.hex
+        URL = "{URL}{key}/".format(URL=self.URL, key=KEY)
+
+        self.client.delete(URL)
+        response = self.client.get(URL)
+
+        self.assertEqual(response.data.get("key").replace("-", ""), KEY)
+        self.assertEqual(response.data.get("status").lower(), "inactive")
+
+    def test_get_invalid_tag_with_filter(self):
+        """Used to test getting a tag given its key."""
+        KEY = self.example_tag.key.hex
+        URL = "{URL}{key}/".format(URL=self.URL, key=KEY)
+        URL_WITH_FILTER = "{URL}?key={key}".format(URL=self.URL, key=KEY)
```

Comment:
you can send the queryparams as a dict, instead of create the string with the query prams

Reply:
I added the queryparams in another PR